src/ctia/events/obj_to_event.clj

```diff
@@ -0,0 +1,61 @@
+(ns ctia.events.obj-to-event
+  (:require [clojure.data :refer [diff]]
+            [schema.core :as s]
+            [ctia.lib.time :as t]
+            [ctia.schemas.actor :refer [StoredActor]]
+            [ctia.schemas.campaign :refer [StoredCampaign]]
+
+            [ctia.events.schemas :as vs]))
+
+
+(s/defn to-create-event :- vs/CreateEvent
+  [model :-  {s/Any s/Any}
```

Comment:
I think this came from me 2 months ago.
It's indicative of a map, but I did not know if the keys were keywords or strings (I haven't seen a model yet, but I'm now presuming that the keys will be keywords). If models have a defined set of keys, then it would obviously be better to use that schema. Otherwise, the key type should be updated